### PR TITLE
Remove PCL dependency

### PR DESCRIPTION
**Pull request by vrabaud**

### CODE REVIEW

File: perception/point_containment_filter/src/impl/point_cloud2_iterator.h [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+ *  modification, are permitted provided that the following conditions
+ *  are met:
+ *
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above
+ *     copyright notice, this list of conditions and the following
+ *     disclaimer in the documentation and/or other materials provided
+ *     with the distribution.
+ *   * Neither the name of Willow Garage, Inc. nor the names of its
```
Comment (sachinchitta) [inline perception/point_containment_filter/src/impl/point_cloud2_iterator.h]: Correct license information

File: perception/point_containment_filter/src/point_cloud2_iterator.h [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+ *  modification, are permitted provided that the following conditions
+ *  are met:
+ *
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above
+ *     copyright notice, this list of conditions and the following
+ *     disclaimer in the documentation and/or other materials provided
+ *     with the distribution.
+ *   * Neither the name of Willow Garage, Inc. nor the names of its
```
Comment (sachinchitta) [inline perception/point_containment_filter/src/point_cloud2_iterator.h]: This should be OSRF?

File: perception/pointcloud_octomap_updater/src/impl/point_cloud2_iterator.h [hunk 3]
```diff
@@ -0,0 +1,436 @@
...
+ *  modification, are permitted provided that the following conditions
+ *  are met:
+ *
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above
+ *     copyright notice, this list of conditions and the following
+ *     disclaimer in the documentation and/or other materials provided
+ *     with the distribution.
+ *   * Neither the name of Willow Garage, Inc. nor the names of its
```
Comment (sachinchitta) [inline perception/pointcloud_octomap_updater/src/impl/point_cloud2_iterator.h]: Again, OSRF or Vincent?

File: perception/pointcloud_octomap_updater/src/point_cloud2_iterator.h [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+ *  modification, are permitted provided that the following conditions
+ *  are met:
+ *
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above
+ *     copyright notice, this list of conditions and the following
+ *     disclaimer in the documentation and/or other materials provided
+ *     with the distribution.
+ *   * Neither the name of Willow Garage, Inc. nor the names of its
```
Comment (sachinchitta) [inline perception/pointcloud_octomap_updater/src/point_cloud2_iterator.h]: One last place to change this.

**Comment by isucan** [comment]
+1
Vincent, thanks for the patch. Should the iterator stuff be a separate package?

**Comment by vrabaud** [comment]
ok, I fixed the licenses.
Those files are in sensor_msgs in Indigo but will not be backported to hydro according to https://github.com/ros/common_msgs/pull/16, hence the copy.

**Comment by davetcoleman** [comment]
@vrabaud it looks like the repo is out of sync with this PR, could you try updating it to the latest hydro-devel?

is there some other reason this has not been merged yet?

**Comment by vrabaud** [comment]
Hey, the Travis job passed. Now, I copied the PointCloud2 iterators over so that it also works in Hydro. I see that you have branched moveit_core for Indigo. Are you going to do the same for moveit_ros ? In which case I would simpify my PR and only make it for Indigo.

**Comment by isucan** [comment]
I think we should branch for indigo.

**Comment by vrabaud** [comment]
Then please do and I'll make my PR there. Thx.

**Comment by davetcoleman** [comment]
I have branched moveit_ros for indigo as requested:
https://github.com/ros-planning/moveit_ros/tree/indigo-devel

Now that moveit_core, moveit_ros, and moveit_planners are branched, I vote we set them as the default github branches. Anyone opposed?

**Comment by isucan** [comment]
+1 for default branch

**Comment by vrabaud** [comment]
thx, closing that in favor of https://github.com/ros-planning/moveit_ros/pull/490

**Comment by davetcoleman** [comment]
moveit_core, moveit_ros, and moveit_planners now default to the indigo branch

**Pull request closed without merging**